src/common/description.cc

```diff
@@ -15,8 +15,14 @@
 #include <pistache/http_header.h>
 
 #include <algorithm>
-#include <filesystem>
 #include <sstream>
+#if __GNUG__ <= 7
+#include <experimental/filesystem>
+namespace filesystem = std::experimental::filesystem;
+#else
+#include <filesystem>
+namespace filesystem = std::filesystem;
+#endif
```

Comment:
This is still problematic. It is still causing build failures in the autopkgtest job, and will also break compilation on Clang, as it always sets the `__GNUG__` macro to `4`.

    $ clang++ -x c++ /dev/null -dM -E | grep GNUG
    #define __GNUG__ 4

You could try doing what I suggested in https://github.com/pistacheio/pistache/commit/e0c3f0f840dea155d67439385aed6613a1d54736#r79759584

```suggestion
#if __has_include(<filesystem>)
#include <filesystem>
namespace fs = std::filesystem;
#elif __has_include(<experimental/filesystem>)
#include <experimental/filesystem>
namespace fs = std::experimental::filesystem;
#endif
```